src/game/World.h

```diff
@@ -449,7 +449,8 @@ enum eConfigFloatValues
     CONFIG_FLOAT_RATE_XP_PERSONAL_MAX,
     CONFIG_FLOAT_AC_MOVEMENT_CHEAT_TELEPORT_DISTANCE,
     CONFIG_FLOAT_AC_MOVEMENT_CHEAT_WALL_CLIMB_ANGLE,
-    CONFIG_FLOAT_VALUE_COUNT
+    CONFIG_FLOAT_VALUE_COUNT,
+    CONFIG_FLOAT_RP_DECAY
```

Comment:
This needs to be above "CONFIG_FLOAT_VALUE_COUNT"

Reply:
Whoops, thanks! Fixed in https://github.com/vmangos/core/commit/bd5841a4b84941f6148a66ac4568f59d9cfd0289.

Reply:
Speaking only for myself, I wasn't familiar with this technique of getting the enum size, though I won't use that as excuse since the naming `CONFIG_FLOAT_VALUE_COUNT` is pretty clear. 😄